components/match2/wikis/sideswipe/opponent_display_custom.lua

```diff
@@ -0,0 +1,87 @@
+---
+-- @Liquipedia
+-- wiki=sideswipe
+-- page=Module:OpponentDisplay/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+
+local DisplayHelper = Lua.import('Module:MatchGroup/Display/Helper', {requireDevIfEnabled = true})
+local OpponentDisplay = Lua.import('Module:OpponentDisplay', {requireDevIfEnabled = true})
+
+local CustomOpponentDisplay = {propTypes = {}, types = {}}
+
+CustomOpponentDisplay.BracketOpponentEntry = Class.new(OpponentDisplay.BracketOpponentEntry, function(self) end)
+
+function CustomOpponentDisplay.BracketOpponentEntry:addScores(opponent)
+	local extradata = opponent.extradata or {}
+	if not extradata.additionalScores then
+		OpponentDisplay.BracketOpponentEntry.addScores(self, opponent)
+	else
+		local score1Node = OpponentDisplay.BracketScore({
+			isWinner = extradata.set1win,
+			scoreText = OpponentDisplay.InlineScore(opponent),
+		})
+		self.root:node(score1Node)
+
+		local score2Node
+		if opponent.extradata.score2 or opponent.score2 then
+			score2Node = OpponentDisplay.BracketScore({
+				isWinner = extradata.set2win,
+				scoreText = CustomOpponentDisplay.InlineScore2(opponent),
+			})
+		end
+		self.root:node(score2Node)
+
+		local score3Node
+		if opponent.extradata.score3 then
+			score3Node = OpponentDisplay.BracketScore({
+				isWinner = extradata.set3win,
+				scoreText = CustomOpponentDisplay.InlineScore3(opponent),
+			})
+		end
+		self.root:node(score3Node)
+
+		if (opponent.placement2 or opponent.placement or 0) == 1
+			or opponent.advances then
+			self.content:addClass('brkts-opponent-win')
+		end
+	end
+end
+
+--[[
+Displays the second score or status of the opponent, as a string.
+]]
+function CustomOpponentDisplay.InlineScore2(opponent)
```

Comment:
should imo be merged with `CustomOpponentDisplay.InlineScore3`
those are identical with the exception of `2` and `3`

Reply:
Agreed